odoo13_install_debian10.sh

```diff
@@ -0,0 +1,359 @@
+#!/bin/bash
+################################################################################
+# Script for installing Odoo on Debian 10.0 (could be used for other version too)
+# Author: Yenthe Van Ginneken
+# Maintainer: César Cordero Rodríguez
```

Comment:
Can you set both of our names on both the author and maintainer lines please? Looks more correct then

Reply:
Done. Thanks.